### PR TITLE
Add in basic action with g8Test

### PR DESCRIPTION
**Pull request by ckipp01**
I figured it might be a good idea just to run `g8Test` with these just to ensure they are working correctly when we make changes. I don't have any matrixes or caches or anything fancy added mainly because this is a super simple template, and the biggest bang is just making sure it works in general. Plus the whole thing just takes a minute to run, so I figured there was no point in the cache.

You can see this on my local master [here](https://github.com/ckipp01/mill-scala-seed.g8/runs/773638851?check_suite_focus=true)

Also as a side note, I did also try this in the gradle one, but am unable to get `g8Test` to pass, so I'll need to spend a bit more time on that one 🤔

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,6 +8,15 @@ lazy val root = (project in file("."))
     test in Test := {
       val _ = (g8Test in Test).toTask("").value
     },
-    scriptedLaunchOpts ++= List("-Xms1024m", "-Xmx1024m", "-XX:ReservedCodeCacheSize=128m", "-Xss2m", "-Dfile.encoding=UTF-8"),
-    resolvers += Resolver.url("typesafe", url("https://repo.typesafe.com/typesafe/ivy-releases/"))(Resolver.ivyStylePatterns)
+    scriptedLaunchOpts ++= List(
```
Comment (ckipp01) [inline build.sbt:11]: Nothing changed here, I just ran scalafmt on it.

**Review by tgodzik — APPROVED** [review]
Looks good! Thanks for adding it! Do we have to enable github specifically for the repository or does it work after first merging the workflow file?

**Comment by tgodzik** [comment]
Running `g8Test` in mill workspace doesn't do anything, because the layout is not standrad. Running it in a more standard directory structure makes sbt want to compile and it fails doing it. We might need to make the scripted test empty in gradle-scala-seed.

**Comment by ckipp01** [comment]
> Running `g8Test` in mill workspace doesn't do anything, because the layout is not standrad. Running it in a more standard directory structure makes sbt want to compile and it fails doing it. We might need to make the scripted test empty in gradle-scala-seed.

Yea, so last night after I did this and signed off it dawned on me that the mill one was passing because it was wasn't really doing anything. Do we still want this if it's not technically "testing" anything?

**Comment by tgodzik** [comment]
> > Running `g8Test` in mill workspace doesn't do anything, because the layout is not standrad. Running it in a more standard directory structure makes sbt want to compile and it fails doing it. We might need to make the scripted test empty in gradle-scala-seed.
> 
> Yea, so last night after I did this and signed off it dawned on me that the mill one was passing because it was wasn't really doing anything. Do we still want this if it's not technically "testing" anything?

It's at least testing applying the template, which is good for the start. We could extend it at some point to actually invoking mill, but I am not sure 100% how to do it.

**Comment by ckipp01** [comment]
> Looks good! Thanks for adding it! Do we have to enable github specifically for the repository or does it work after first merging the workflow file?

Ah, sorry forgot to answer this. No need to enable it, it will just go.
